src/main/resources/assets/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/leggings/tconstruct_crafting_table.json

```
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:crafting_table",
+  "text": [
+    {
+      "text": "就像你口袋里所有的线都缠成一团，但却很有用。"
```

```suggestion
      "text": "就像你口袋里所有的线都缠成一团一样，但却很有用。"
```
Polishing, not required.